syft/serde/serde.py

```diff
@@ -40,14 +36,15 @@ def serialize(
     Returns:
         binary: the serialized form of the object.
     """
+    if strategy is None:
+        from syft.serde.msgpack import serialize
```

Comment:
Is this here to avoid some kind of circular imports?

Reply:
Yes, as a temporary solution, we'll leave it here and figure out a solution to break those circular changes in a more elegant way.